### PR TITLE
Getlog

### PR DESCRIPTION
**Pull request by yifan-gu**
@vmarmol @dchen1107 @jonboulle @bakins

This adds the `GetContainerLogs` for rkt.

Also follow #7465

### CODE REVIEW

**Comment by vmarmol** [comment]
LGTM, minor comment.

**Comment by yifan-gu** [comment]
OK, let's do UID. Will add a todo to take care of readability.

File: pkg/kubelet/rkt/rkt.go [hunk 1]
```diff
@@ -142,3 +143,9 @@ func (r *Runtime) runCommand(args ...string) ([]string, error) {
 	}
 	return strings.Split(strings.TrimSpace(string(output)), "\n"), nil
 }
+
+// makePodServiceFileName constructs the unit file name for a pod using its UID.
+func makePodServiceFileName(uid types.UID) string {
+	// TODO(yifan): Revisit this later, decide whether we want to use UID.
+	return fmt.Sprintf("%s_%s.service", kubernetesUnitPrefix, name, namespace, uid)
```
Comment (vmarmol) [inline pkg/kubelet/rkt/rkt.go]: Too many args to Sprintf

Comment (yifan-gu) [inline pkg/kubelet/rkt/rkt.go]: Good catch!

**Comment by vmarmol** [comment]
LGTM

**Pull request merged**